viral/offline_sequence_detection.py

```
@@ -0,0 +1,857 @@
+    r_shuffled = np.abs(shuffled_weighted_rs)
+    empirical_p = np.mean(r_shuffled >= r_real)
+    rz_score = (r_real - np.mean(r_shuffled)) / np.std(r_shuffled)
+    print(f"empirical p-value: {empirical_p:.2f}, rZ score: {rz_score:.2f}")
+    return empirical_p < significance
+    # TODO: they use Radon whatever, do we need it as well? (I think because of using the absolute values, the direction isn't correctly accounted for)
+    return
+
+
+def plot_pse_event(
+    posterior_probability_matrix: np.ndarray,
+    pr_max: np.ndarray,
```

Ruff (0.14.5)

Unused function argument: `pr_max`

(ARG001)